src/Tickets/Commerce/Abstract_Order.php

```diff
@@ -95,7 +95,7 @@ public function get_purchaser_data( $data ) {
 		if ( is_user_logged_in() ) {
 			$user                              = wp_get_current_user();
 			$purchaser['purchaser_user_id']    = $user->ID;
-			$purchaser['purchaser_full_name']  = $user->first_name . ' ' . $user->last_name;
+			$purchaser['purchaser_full_name']  = trim( $user->first_name . ' ' . $user->last_name );
```

Comment:
If we need to trim this, do we need to trim the next two as well?

Reply:
we are doing the trim here because of the empty space added regardless of the last_name value. That's not the case for the rest :D